authors/apps/authentication/tests/test_validations.py

```diff
@@ -65,14 +65,4 @@ def test_short_password(self):
         self.assertEqual(errors['password'][0],
                          "Password must be at least 8 characters long")
         self.assertEqual(
-            errors['password'][1], "Password must have alphanumeric characters with no space")
-
-    def test_non_alphanumeric_password(self):
-        """
-        Test user signup with the special characters as password
-        """
-        response = self.signup_user("test", "test@gmail.com", "@@@@@@@@@@")
-        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
-        errors = response.data['errors']
-        self.assertEqual(
-            errors['password'][0], "Password must have alphanumeric characters with no space")
+            errors['password'][1], "Password must have three lowercase, two lowercase, one special case letters and two digits eg 12aaaAA@")
```

Comment:
Hello @Philipotieno ,
In this error message, you have the lowercase policy repeated. In addition, to make the system forgiving enough and password memorable, we could use this policy:
```
a. Minimum of 2 uppercase letters
b. Minimum of 2 special characters
c. Minimum of 2 digits
d. Minimum length of 8
```
With this, the password strengths would be strong and also memorable

Reply:
Ogutu I have corrected that